restapi/api.go

```diff
@@ -23,9 +25,20 @@ func Router(db *gorm.DB) (http.Handler, error) {
 		logger:   zap.S(),
 	}
 
-	rest.GetHandler(r, "/health", api.health)
-	rest.GetHandler(r, "/stories", api.ReturnAllStories)
-	rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
+	// Public API
+	r.Group(func(r chi.Router) {
+		rest.GetHandler(r, "/health", api.health)
+		rest.GetHandler(r, "/login/{username}/{password}", api.Login)
+		rest.GetHandler(r, "/stories", api.ReturnAllStories)
+		rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
+	})
+	
+	// Private API
+	r.Group(func(r chi.Router) {
+		r.Use(middleware.VerifyToken)
```

Comment:
`chi` has a middeware for this already. I think we should try using that instead of creating our own!

https://github.com/go-chi/jwtauth

Reply:
🤦‍♀️ will swap out